src/pages/admin/reports/components/program-expenses-section/components/program-expenses-empty-state/ProgramExpensesEmptyState.module.scss

```
@@ -0,0 +1,124 @@
+.empty-state-image {
+    width: 180px;
+    height: auto;
+}
+
+.empty-state-image-program-expenses {
+    margin-bottom: 22px;
+}
+
+.empty-state-title {
+    @include type.h4-semibold;
+    line-height: 24px;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)